theano/sandbox/gpuarray/blas.py

```diff
@@ -262,19 +283,18 @@ def c_code(self, node, name, inputs, outputs, sub):
         return code
 
     def c_code_cache_version(self):
-        return (1,)
+        return (2,)
+
+    def c_header_dir(self):
+        ret = super(GpuDot22, self).c_header_dirs()
```

Comment:
This isn't good. It can raise MethodNotImplemented and loose the new information.